core/player/src/view/view.ts

```diff
@@ -94,21 +96,29 @@ export class ViewInstance implements ValidationProvider {
   // TODO might want to add a version/timestamp to this to compare updates
   public lastUpdate: Record<string, any> | undefined;
 
-  constructor(initialView: ViewType, resolverOptions: Resolve.ResolverOptions) {
+  constructor(initialView: ViewType, resolverOptions: Resolve.ResolverOptions & ViewControllerOptions) {
     this.initialView = initialView;
     this.resolverOptions = resolverOptions;
     this.hooks.onTemplatePluginCreated.tap("view", (templatePlugin) => {
       this.templatePlugin = templatePlugin;
     });
+    resolverOptions.flowController.current?.hooks.transition.tap("view", (state1, state2) => {
+      this.transitioning = true;
+    })
+    resolverOptions.flowController.current?.hooks.afterTransition.tap("view", (flowInstance) => {
+      this.transitioning = false;
+    })
```

Comment:
You'll want to tap the `flow` hook on the `flowController` instead of using `current`